docs/server/push-duty.rst

```diff
@@ -167,9 +181,20 @@ The tag is pushed to production by ops (wezhou), once approved by QA (Krupa), on
 It is the responsibility of the push hero to follow-up with QA and ops,
 and be around during the push for any unexpected issues.
 
+Extension Workshop
+++++++++++++++++++
```

Comment:
`-------` instead?

Reply:
That would be incorrect. This is meant to be a sub-heading under `Push`, which already uses `----` so using that again here would make it a sibling to `Push` as opposed to a sub-heading.

Reply:
I meant `_____`, like the other section under `Tag the repos`, but I see that's at a different level (the doc is pretty unbalanced, imo, but meh)